cypress/integration/components/categoy-card.spec.js

```diff
@@ -0,0 +1,21 @@
+describe('Teste de comportamento do componente categoryCard', () => {
+    it('True será retornado se o componente está visível antes de ser clicado ', () => {
+        cy.visit('/')
+        cy.get('.style_categoryCard__2FPBJ').should('be.visible')
+        cy.get('.style_container__3S9Rh').should('be.visible')
```

Comment:
futuramente
```suggestion
        cy.visit('/')
        cy.contains('Categorias').should('be.visible').click()  // esse clica no link de categorias através do nome, por enquanto não possui nenhum id específico para ele, aconselho a colocar para pegar através do cy.get('#...').
        cy.get('.style_categoryCard__2FPBJ').should('be.visible')
        cy.get('.style_container__3S9Rh').should('be.visible')
    
```
dessa forma, navega para a página de categorias

Reply:
mesma alteração para o de baixo